### ISSUE TITLE
Remove zoom buttons on Android webview?

### ISSUE THREAD

**Issue by eteled126** [peiche/recover#42]
Hi, this is the first screen to open my website, on an Android phone:
<img src="https://s2.ax1x.com/2019/02/13/k05Jx0.jpg" alt="k05Jx0.jpg" border="0">

I noticed that there is a zoom button in the lower right corner. In fact, it's almost always there, as long as you touch or slide the screen (even if there is no zooming action).

It's too conspicuous and not practical, I want to eliminate it, or hide it.

<img src="https://s2.ax1x.com/2019/02/13/k05tMV.jpg" alt="k05tMV.jpg" border="0">

I found that in Chrome, this button doesn't appear. Except for this, it appears in any other app. It exist when you use the apps who with built-in browsers to access the site. (The screenshot of the app is WeChat)

Most of the sites I visit regularly, no such button appears. Is the cover2 theme not suitable for mobile devices?

<img src="https://i.loli.net/2019/02/13/5c640c18992a0.jpg" alt="WeChat Image_20190213201802.jpg" title="WeChat Image_20190213201802.jpg" />

**Comment by peiche** [comment]
The zoom buttons are controlled by a combination of a couple things. First, whether the webview in an app enables them. Obviously, that's out of the control of _any_ website. Second, whether the website in question allows zoom. This is Cover2's meta viewport tag:

```
<meta name="viewport" content="width=device-width, initial-scale=1">
```

This is standard for any responsive site. If I were to disable zoom, I would change it to this:

```
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

The properties `maximum-scale=1, user-scalable=no` make sure the largest available zoom is 100%, and disable pinch to zoom. This would remove the zoom buttons in any app's webview, but it would also disable zoom, which I don't want to do.

TL;DR there's nothing I can do. And yes, Cover2 is suitable responsive theme.

**Comment by teusbenschop** [comment]
The code of the Android app was updated as follows:
```
webview.getSettings().setBuiltInZoomControls (false);
```

Then an update was release to the Beta channel on the Google Play Store.
This Beta app was tested on the Nexus 6.
But the affected links remained unclickable.

The conclusion is that this code updated that not resolve the problem.

I am leaving the code update in because it's good to get rid of the zoom controls that at times get in the way anyway.

So although the code update didn't fix the problem, it's still go to have it.

**Comment by teusbenschop** [comment]
The Android WebView API at
https://developer.android.com/reference/android/webkit/WebSettings.html#setBuiltInZoomControls(boolean)
confirms that the built-in zoom controls cover content on the WebView:

```
public abstract void setBuiltInZoomControls (boolean enabled)
```
Sets whether the WebView should use its built-in zoom mechanisms. The built-in zoom mechanisms comprise on-screen zoom controls, which are displayed over the WebView's content, and the use of a pinch gesture to control zooming. Whether or not these on-screen controls are displayed can be set with setDisplayZoomControls(boolean). The default is false.

The built-in mechanisms are the only currently supported zoom mechanisms, so it is recommended that this setting is always enabled.

**Comment by teusbenschop** [comment]
Sorry I am posting in the wrong issue tracker...
